query.go

```diff
@@ -1096,6 +1098,18 @@ func (s *ChainService) sendTransaction(tx *wire.MsgTx, options ...QueryOption) e
 					response, sp.Addr(),
 				)
 				rejections[*broadcastErr]++
+
+				// To ever get (numReplied == len(rejections)), it is necessary
+				// to also increment numReplied. However, this would be a
+				// breaking change since common innocuous errors such as Mempool
+				// and Confirmed do not presently cause a non-nil error return.
+				//
+				// numReplied++
```

Comment:
It is.  This is the bit in the PR description regarding @wpaulino (wilmer)'s comments.  The conditional below is impossible to hit afaict, and this commented change would seem to be the intention.  However, it's a breaking change since it would start returning errors in harmless situations such as the one described here and below w.r.t the `pushtx` types.

Reply:
see my previous comment. I think we can just account for those non-critical error cases